src/maidsafe/common/tests/convert_test.cc

```diff
@@ -0,0 +1,79 @@
+/*  Copyright 2009 MaidSafe.net limited
+
+    This MaidSafe Software is licensed to you under (1) the MaidSafe.net Commercial License,
+    version 1.0 or later, or (2) The General Public License (GPL), version 3, depending on which
+    licence you accepted on initial access to the Software (the "Licences").
+
+    By contributing code to the MaidSafe Software, or to this project generally, you agree to be
+    bound by the terms of the MaidSafe Contributor Agreement, version 1.0, found in the root
+    directory of this project at LICENSE, COPYING and CONTRIBUTOR respectively and also
+    available at: http://www.maidsafe.net/licenses
+
+    Unless required by applicable law or agreed to in writing, the MaidSafe Software distributed
+    under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+    OF ANY KIND, either express or implied.
+
+    See the Licences for the specific language governing permissions and limitations relating to
+    use of the MaidSafe Software.                                                                 */
+
+#include "maidsafe/common/log.h"
+#include "maidsafe/common/test.h"
+#include "maidsafe/common/convert.h"
+
+namespace maidsafe {
+
+namespace test {
+
+struct Conversions : public ::testing::Test {
+  virtual void SetUp() { }
+  virtual void TearDown() { }
+};
+
+template<class T> void ToBoostThenBack(const T& value) {
+  auto boost_value = common::convert::ToBoost(value);
+  auto second_value = common::convert::ToAsio(boost_value);
+  EXPECT_EQ(value, second_value);
+}
+
+template<class T> void ToAsioThenBack(const T& value) {
+  auto boost_value = common::convert::ToAsio(value);
+  auto second_value = common::convert::ToBoost(boost_value);
+  EXPECT_EQ(value, second_value);
+}
+
+TEST_F(Conversions, BEH_Address_v4) {
+  const char* ip = "123.231.213.213";
```

Comment:
Could you use the [`GetRandomIPv4AddressAsString()`](https://github.com/Fraser999/MaidSafe-Common/blob/next/include/maidsafe/common/test.h#L55) helper function in test.h here please so we're not using a hard-coded value?